### PR TITLE
Add `ExpandAttribute` handler

### PR DESCRIPTION
**Pull request by Draggu**
**Stack**:
- #1686
- #1685
- #1684 ⬅

⚠️ *Part of a stack created by [spr](https://github.com/ejoffe/spr). Do not merge manually using the UI - doing so may have unexpected results.*

### CODE REVIEW

File: scarb/src/ops/proc_macro_server/methods/expand_attribute.rs
Comment (mkaput) [inline scarb/src/ops/proc_macro_server/methods/expand_attribute.rs:1]: missing tests

**Review by mkaput — CHANGES_REQUESTED** [review]

File: scarb/tests/proc_macro_server.rs [hunk 1]
```diff
@@ -36,3 +39,37 @@ fn defined_macros() {
...
+    let response = proc_macro_server
+        .request_and_wait::<ExpandAttribute>(ExpandAttributeParams {
+            attr: "some".to_string(),
+            args: TokenStream::empty(),
+            item: item.clone(),
+        })
+        .ok();
+
+    assert_eq!(response.diagnostics, vec![]);
+    assert_eq!(response.token_stream, item);
```
Comment (piotmag769) [inline scarb/tests/proc_macro_server.rs:74]: Can we do anything in the macro so this test actually tests if the logic executes?

Comment (mkaput) [reply scarb/tests/proc_macro_server.rs:74]: I think asserting on the debug-print will be enough

**Review by mkaput — CHANGES_REQUESTED** [review]
blocking due to Piotr's comment

File: scarb/tests/proc_macro_server.rs (hunk 1 above)
Comment (Draggu) [reply scarb/tests/proc_macro_server.rs:74]: >I think asserting on the debug-print will be enough

Will not as these are not proc-macros

**Review by piotmag769 — APPROVED** [review]

**Pull request merged**